Database/Stored Procedures/Add_or_Update/add_visitor_visited_performance.sql

```diff
@@ -0,0 +1,50 @@
+/*==============================================================*/
+/* DBMS name:		FestiBase									*/
+/* PDM version:		6											*/
+/* Last edited:		08-01-2018									*/
+/* Edited by:		Mariusz Blautzik							*/
+/* Procedure:		Insert VISITOR VISITED PERFORMANCE			*/
+/*==============================================================*/
+
+USE FestiBase
+GO
+
+/*
+	Procedure to add a VISITOR VISITED PERFORMANCE
+*/
+DROP PROC IF EXISTS sp_add_visitor_visited_performance
+GO
+CREATE PROC sp_add_visitor_visited_performance
+	@visitor_number				INT,
+    @performance_number			INT
+AS
+BEGIN
+	BEGIN TRY
+			BEGIN
+				INSERT INTO VISITOR_visited_PERFORMANCE(visitor_number, performance_number) VALUES
+				(@visitor_number,
+				 @performance_number
+				 )
+			END
+	END TRY
+	BEGIN CATCH
+		;THROW 50000, '@visitor_number + @performance_number does not match .', 1 
```

Comment:
de enige manier hoe deze insert fout kan gaan is door een nummer te inserten die niet bestaat, misschien error message aanpassen naar " @visitor_number or @performance_number does not exist. "